client/debug.log

```diff
@@ -1 +1,2 @@
 [1017/021103.412:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
+[1017/133118.740:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
what is this file? do we need this?

Reply:
i dontthink to, I have deleted one before but since it popped up I didnt want to try and sweep it under the rug again, may be just some standard errors that create there own log

Reply:
It's useful if we know what causes it, but I don't think we need to keep it in the repo if that makes sense. Thank you for deleting 👍